src/types.ts

```diff
@@ -37,11 +54,10 @@ export interface ExecutableSchemaArgs<TContext = any> {
   resolvers?: IResolvers<any, TContext> | Array<IResolvers<any, TContext>>; // resolvers for the type definitions
 }
 
-type Maybe<T> = null | undefined | T;
+export type Maybe<T> = null | undefined | T;
```

Comment:
What is this type here? Maybe<T>?

Reply:
It is a generic type. The type given or its going to be null or undefined.